src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -415,6 +416,28 @@ internal async Task ReceiveByteArray(int id, byte[] data)
             }
         }
 
+        // SupplyJSDataChunk is used in a fire-and-forget context, so it's responsible for its own
+        // error handling.
+        internal async Task<bool> SupplyJSDataChunk(string streamId, ReadOnlySequence<byte> chunk, string error)
```

Comment:
The naming makes me wonder if I've got it backwards or if the code has it backwards. Since this is the JS to .NET direction, isn't it similar to the `ReceiveByteArray` endpoint above? If that one is called *Receive*, how come this one is called *Supply*?

Reply:
Thanks that's a good point, `ReceiveJSDataChunk` would be more in line.